api/createTournament.js

```diff
@@ -7,30 +8,22 @@ const games = [
   'Dota 2',
   'Rocket League',
   'Battalion 1944',
-  'League of Legends'
+  'League of Legends',
 ];
 
 function createTournament(name) {
   return {
-    id: faker.random.uuid(),
+    id: faker.datatype.uuid(),
     name:
       name ||
-      faker.lorem
-        .words(random(2, 4))
-        .split(' ')
-        .map(upperFirst)
-        .join(' '),
-    organizer: faker.lorem
-      .words(random(1, 2))
-      .split(' ')
-      .map(upperFirst)
-      .join(' '),
+      faker.lorem.words(random(2, 4)).split(' ').map(upperFirst).join(' '),
+    organizer: faker.company.companyName(0),
```

Comment:
Do we need to do the same for `name`, as it is using the same approach as `organizer` was before this change?

Reply:
Also, looking at the example that prompted this change, I'm not sure it's really necessary, the `faker.lorem.words` is using [Lorem Ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum), which uses a known list of Latin words, `cum` specifically meaning `with`, for example in the US, the phrase "magna cum laude" is used to mean "with great honours" when graduating from University.

As such, I'm not sure doing anything to avoid offensive language in the generated words is necessary, since we can see a list of all the available words it can generate, and the only one that could be considered "questionable" is `cum`, which does have a legitimate meaning in the context of all the other generated words being latin anyway

Reply:
fair enough 😄 
will revert it back
thanks!

Reply:
done